eslint-rules/src/rules/use-correct-component-overrides.ts

```diff
@@ -0,0 +1,136 @@
+import { Selectors } from '@angular-eslint/utils';
+import { ASTUtils, TSESLint, TSESTree } from '@typescript-eslint/experimental-utils';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { normalizePath } from '../helpers';
+
+export const useCorrectComponentOverridesRule: TSESLint.RuleModule<string, []> = {
```

Comment:
Please add some documentation to explain the rule, what it does and how to configure it.